docs/getting-started/server/guide-vscode.md

```diff
@@ -0,0 +1,270 @@
+---
+sidebar_position: 2
+---
+
+# VS Code Dev Containers
+
+This page will show you how to set up a local Bitwarden server for development purposes utilizing
+Visual Studio Code Dev Containers. You can read more about this in the
+[Visual Studio Code Dev Containers documentation](https://code.visualstudio.com/docs/devcontainers/containers).
+
+The Bitwarden server is comprised of several services that can run independently. For a basic
+development setup, you will need the **Admin**, **Api**, and **Identity** services.
+
+:::info
+
+Before you start: make sure you’ve installed the recommended
+[Tools and Libraries](../tools/index.md). Namely, the following:
+
+- Docker Desktop
+- [Visual Studio Code](https://code.visualstudio.com/)
+- Azure Data Studio
+
+:::
+
+## Clone the repository
+
+1.  Clone the Bitwarden Server project:
+
+    ```bash
+    git clone https://github.com/bitwarden/server.git
+    ```
+
+2.  Open a terminal and navigate to the root of the cloned repository.
+
+:::caution
+
+When first opening the repository in Visual Studio Code, you will be prompted to reopen the folder
+in a Dev Container. If you have not completed the steps in the
+[Configure Env File](#configure-env-file) section below, the creation of the Dev Container will
+fail.
+
+:::
+
+## Configure Git
+
+1. Configure Git to ignore the Prettier revision:
+
+   ```bash
+   git config blame.ignoreRevsFile .git-blame-ignore-revs
+   ```
+
+2. _(Optional)_ Set up the pre-commit `dotnet format` hook:
+
+   ```bash
+   git config --local core.hooksPath .git-hooks
+   ```
+
+   Formatting requires a full build, which may be too slow to do every commit. As an alternative,
+   you can run `dotnet format` from the command line when convenient (e.g. before requesting a PR
+   review).
+
+## Configure Env File
```

Comment:
💭 Can we start this documentation right here? The prior two steps are a repeat of the main setup guide.